pull_request_template.md

```diff
@@ -8,13 +8,19 @@
 [Discuss the impact of your changes on the project. This might include effects on performance, new dependencies, or changes in behaviour.]
 
 ## Checklist
-- [ ] My code adheres to the coding and style guidelines of the project.
-- [ ] I have performed a self-review of my own code.
-- [ ] I have extended current unit tests suite.
-- [ ] I have made corresponding changes to the documentation.
-- [ ] My changes generate no new warnings or new Sonar issues
-- [ ] I have passed the PR linting stage
-- [ ] I contacted my SRE colleague to update the configuration (if any changes)
+- [ ] **[JIRA]** I have linked the JIRA ticket to the PR.

```

Comment:
do we need a checkbox if there is section just above?

Reply:
The ticket needs to be linked by including the ticket in the title and/or description and I would be in favor of eliminating this checkbox in the case where we have automated checks for linking, but an automated check for checking the linkage to JIRA is a bad idea for an SRE member who doesn't need a ticket to change something critical in our repo, therefore I added the checkbox.

Reply:
Exactly, the above point (line 4) is just a text without any enforcing, like that we allow SRE members or security to create a pr without having to link it to a text. We plan for the future to have an action that will not pass if the checkboxes are not checked, like that, SRE members will never merge if they don't have a jira ticket